Makefile

```diff
@@ -3,8 +3,7 @@ include $(THEOS)/makefiles/common.mk
 
 export FINALPACKAGE = 1
 DEBUG = 0
-export ARCHS = arm64 #arm64e
-export TARGET = iphone:clang:13.5
```

Comment:
Necessary to compile with the right SDK using GitHub workflow. You can download mine from [lgariv/theos](https://www.github.com/lgariv/theos)